src/addressResolvers/snapshot.ts

```diff
@@ -33,6 +34,16 @@ export async function lookupAddresses(addresses: Address[]): Promise<Record<Addr
   }
 }
 
-export async function resolveNames(): Promise<Record<Handle, Address>> {
-  return {};
+export async function resolveName(name: string): Promise<Address | undefined> {
```

Comment:
You shouldn't replace `snapshot.ts` resolver but instead making a new file `spaceid.ts`

Reply:
I got it.